### ISSUE TITLE
Write a documentation page on creating good-looking 3D scenes

### ISSUE THREAD

**Issue by Calinou** [godotengine/godot-docs#5290]
See also https://github.com/godotengine/godot-docs/issues/5121.

Looking at various social media, I keep seeing a lot of 3D scenes that don't look great and could easily be made to look significantly better with a little more work. For instance, anisotropic filtering on textures not being enabled in a desktop game.

In the long term, this relatively "amateur" appearance tends to give a bad image of Godot's 3D capabilities. This image may persist after 4.0 if people don't step up their 3D scene setup skills :slightly_smiling_face:

However, I'm worried about this page not being general-purpose enough for the Godot documentation. There's a lot of engine-agnostic resources on creating good-looking 3D scenes, but also a lot of good Unity/Unreal Engine-oriented resources that Godot users may shy away from.

This would be similar to the [Performance](https://docs.godotengine.org/en/latest/tutorials/performance/index.html) section we have, except that it'd be focusing on improving visuals instead. Sometimes, there would be performance tradeoffs involved in improving the visuals, but not always.

**Comment by clayjohn** [comment]
I wonder if this would suit a series of blog posts better. Visual appearance is highly subjective and varies dramatically from game-to-game. Plus as issues are resolved and new features are added, the doc would quickly become out of date and would be a nightmare to maintain.

**Comment by skyace65** [comment]
If people aren't already looking over the various pages to see how lighting and materials should be setup, I doubt they're going to bother looking at this new page. That said, the 3D pages do need to be gone through and cleaned up. I think that would help creators significantly.

**Comment by paddy-exe** [comment]
@skyace65 I disagree. If you have an article on better lighting with a specific demo project, I think it would help a lot of people to get more familiar with adjusting their scenes. Also having an article instead of the docs would still help a lot since it could still be parsed by search engines when searching "How do I improve lighting?" or  "How do I make my game look good?"
